contrib/brl/bseg/bsgm/bsgm_prob_pairwise_dsm.cxx

```diff
@@ -13,13 +13,32 @@
 #include <bvgl/bvgl_k_nearest_neighbors_3d.h>
 #include <bpgl/algo/bpgl_heightmap_from_disparity.h>
 #include <bpgl/algo/bpgl_gridding.h>
-  
 void bsgm_prob_pairwise_dsm::compute_byte()
 {
+<<<<<<< HEAD
```

Comment:
There is a merge conflict here.

Reply:
@hjmjohnson - the merge conflict was an oversight fixed in a later commit (in this same PR).  I updated the branch to eliminate the conflict from the commit history.